applications/forms.py

```diff
@@ -10,29 +10,52 @@
-               'placeholder': 'https://github.com/johnBiene'}))
-    devpost = forms.CharField(required=False, widget=forms.TextInput(
-        attrs={'class': 'form-control',
-               'placeholder': 'https://devpost.com/JohnBiene'}))
-    linkedin = forms.CharField(required=False, widget=forms.TextInput(
-        attrs={'class': 'form-control',
-               'placeholder': 'https://www.linkedin.com/in/john_biene'}))
-    site = forms.CharField(required=False, widget=forms.TextInput(
-        attrs={'class': 'form-control', 'placeholder': 'https://biene.space'}))
-    phone_number = forms.CharField(required=False, widget=forms.TextInput(
-        attrs={'class': 'form-control', 'placeholder': '+#########'}))
-    university = forms.CharField(required=True,
-                                 label='What university do you study at?',
-                                 help_text='Current or most recent school you attended.',
-                                 widget=forms.TextInput(
-                                     attrs={'class': 'typeahead-schools', 'autocomplete': 'off'}))
-
-    degree = forms.CharField(required=True, label='What\'s your major/degree?',
-                             help_text='Current or most recent degree you\'ve received',
-                             widget=forms.TextInput(
-                                 attrs={'class': 'typeahead-degrees', 'autocomplete': 'off'}))
+    github = forms.CharField(
+        required=False,
+        widget=forms.TextInput(
+            attrs={'class': 'form-control',
+                   'placeholder': 'https://github.com/johnBiene'})
+    )
+
+    devpost = forms.CharField(
+        required=False,
+        widget=forms.TextInput(
+            attrs={'class': 'form-control',
+                   'placeholder': 'https://devpost.com/JohnBiene'})
+    )
+
+    linkedin = forms.CharField(
+        required=False,
+        widget=forms.TextInput(
+            attrs={'class': 'form-control',
+                   'placeholder': 'https://www.linkedin.com/in/john_biene'})
+    )
+
+    site = forms.CharField(
+        required=False,
+        widget=forms.TextInput(
+            attrs={'class': 'form-control', 'placeholder': 'https://biene.space'})
+    )
+
+    phone_number = forms.CharField(
+        required=False,
+        widget=forms.TextInput(
+            attrs={'class': 'form-control', 'placeholder': '+#########'})
+    )
+
+    university = forms.CharField(
+        required=True,
+        label='What university do you study at?',
+        help_text='Current or most recent school you attended.',
+        widget=forms.TextInput(
+            attrs={'class': 'typeahead-schools', 'autocomplete': 'off'}))
+
+    degree = forms.CharField(
+        required=True,
+        label='What\'s your major/degree?',
+        help_text='Current or most recent degree you\'ve received',
+        widget=forms.TextInput(
+            attrs={'class': 'typeahead-degrees', 'autocomplete': 'off'}))
```

Comment:
Same here. No idea what has changed here bc you changed the style of the code. Do not change the style of the code pls.